src/filter.js

```diff
@@ -15,19 +15,21 @@
  */
 
 const arrify = require('arrify');
-const createErrorClass = require('create-error-class');
 const extend = require('extend');
 const escapeStringRegexp = require('escape-string-regexp');
 const is = require('is');
 
-const Mutation = require('./mutation.js');
+const Mutation = require('./mutation');
 
 /**
  * @private
  */
-const FilterError = createErrorClass('FilterError', function(filter) {
-  this.message = `Unknown filter: ${filter}.`;
-});
+class FilterError extends Error {
```

Comment:
The one issue with this is we lose the `.name` property on the function, which breaks this (possibly documented) behavior:

```js
if (err.name === 'FilterError') {
 // handle filter error
}
```

Reply:
Fixed, good catch. I've updated the other ones too